packages/bierzo-wallet/src/routes/payment/index.tsx

```diff
@@ -70,7 +70,7 @@ const Payment = (): JSX.Element => {
       if (transactionId === null) {
         toast.show('Request rejected', ToastVariant.ERROR);
       } else {
-        toast.show(`Transaction successful with ID: ${transactionId.slice(0, 10)}...`, ToastVariant.SUCCESS);
+        history.push(BALANCE_ROUTE);
```

Comment:
Oh, now we get no confirmation at all. See also https://github.com/iov-one/ponferrada/issues/415

I think we should not change this code before #415

Reply:
ok

Reply:
I think it is better to leave it. Because we will need to implement redirect to confirmation page and we already have test for this.

Reply:
Fair point 👍

Reply:
But after thinking a little and after reading this [comment](https://github.com/iov-one/ponferrada/pull/417#discussion_r308111002) I decided to remove it.